### ISSUE TITLE
Magic Forms plugin not installing on Build 420

### ISSUE THREAD

**Issue by markandmedia** [skydiver/october-plugin-forms#74]
Probably already a known issue, during a test drive with Build 420 (Laravel 5.5 update), the Magic Forms plugin does not install.

I love your plugin! This is of course no problem at the moment

**Comment by Alex360hd** [comment]
Personnaly, I could install magic form on 420, but it thow a fatal error on all pages (don't know if it's better) ^^

[Here is an attached screenshot of the error it throw](https://ibb.co/chQVnQ)

**Comment by skydiver** [comment]
Will be fixed in the next version

**Comment by skydiver** [comment]
@markandmedia or @Alex360hd 

The update is on the master branch, can anyone test if it's fixed on 420?

Thanks.-

**Comment by markandmedia** [comment]
I very well might be doing something wrong,. 

Downloaded the plugin from GitHub and uploaded this version in the test-site. There is nog error when I load the page with the component, but after a submit it throws the following sql error:

"SQLSTATE[42S02]: Base table or view not found: 1146 Table 'frisselstein_db.martin_forms_records' doesn't exist (SQL: insert into `martin_forms_records` (`ip`, `form_data`, `group`, `updated_at`, `created_at`) values (86.91.61.161, {"Naam":"Test naam","Straatnaam":"volkelseweg","Nummer":"8","Postcode":"5405NA","Woonplaats":"Uden","email":"markvandenelzen@gmail.com","Bericht":"test"}, Contact formulier, 2017-08-29 13:36:13, 2017-08-29 13:36:13))" on line 647 of /var/www/vhosts/markand.nl/frisselstein.markand.nl/vendor/laravel/framework/src/Illuminate/Database/Connection.php

**Comment by skydiver** [comment]
@markandmedia have you logged in and logged out in the backend?

(when logged in the magic forms tables are created)

**Comment by markandmedia** [comment]
I tried that but during login in it throws the following error:

![screenshot](https://user-images.githubusercontent.com/10975290/29824591-3deb69e8-8cd2-11e7-8db4-7966f1ea381c.jpg)

**Comment by Alex360hd** [comment]
Personnally I could update october to 420 and it works with magic form. You have to update magic form **before** october CMS.

Don't forget to launch the following command line :

    php artisan october:up

to force magic form to update table while stile on 419, then, upgrade october to 420.

**Comment by skydiver** [comment]
The update was pushed to "edge channel" ... you can update directly from october backend

**Comment by markandmedia** [comment]
Finally had some time to do a new test. I first installed a clean copy of octoberCMS, installed this plugin and after those steps did the Edge update (build 421).

The errors are now gone! There is one error left after sending the form:

"Address in mailbox given [] does not comply with RFC 2822, 3.6.2." on line 354 of /vendor/swiftmailer/swiftmailer/lib/classes/Swift/Mime/Headers/MailboxHeader.php"



Plugin settings as written after configuring the plugin from the octoberCMS:

[genericForm]
group = "Contact formulier"
rules[Naam] = "required"
rules[Email] = "required|email"
messages_success = "Bedankt voor het invullen, we hebben het formulier met succes ontvangen!"
messages_errors = "Er is helaas iet smis gegaan bij het versturen van het formulier"
mail_enabled = 1
mail_subject = "Contact formulier"
mail_recipients[] = "mark@markandmedia.nl"


As a solution removing [] works (I did not try adding a second email address, it might work with multiple addresses out of the box):

mail_recipients = "mark@markandmedia.nl"


A small thing, probably easily fixed in a future update!

Once again, I'm truly happy with this plugin you created!! There is no way I will be using a Edge version of octoberCMS in a production setting.
